docs/guide.rst

```diff
@@ -122,6 +134,16 @@ Short code example how to use it:
     :language: python
     :dedent: 4
 
+Deploying and using deployed contract in the same transaction
+#############################################################
+
+:ref:`Deployer` is designed to work with multicalls too. It allows to deploy a contract
+and call its methods in the same multicall. Isn't it brilliant? Check out the code!
```

Comment:
"multicall, ensuring atomicity of all operations combined." or something similar? I want to emphasise the benefit.